### PR TITLE
fix(docs): compute & console nav button maj & other corrections MTA-1563

### PR DESCRIPTION
**Pull request by RoRoJ**
Part 1 of MTA-1563 (Console & Compute)

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: compute/kubernetes/how-to/edit-cluster.mdx [hunk 1]
```diff
@@ -36,6 +36,6 @@ The following options are available to see more information about your cluster a
 * **Delete**: Delete the cluster
 
 <Navigation title="See Also">
-  <PreviousButton to="/compute/kubernetes/how-to/create-cluster/">How to create a Kapsule Cluster</PreviousButton>
-  <NextButton to="/compute/kubernetes/how-to/create-kosmos-cluster/">How to create a Kosmos Cluster</NextButton>
+  <PreviousButton to="/compute/kubernetes/how-to/create-cluster/">How to create a Kapsule cluster</PreviousButton>
+  <NextButton to="/compute/kubernetes/how-to/create-kosmos-cluster/">How to create a Kosmos luster</NextButton>
```
Comment (ldecarvalho-doc) [inline compute/kubernetes/how-to/edit-cluster.mdx:40]:
```suggestion
  <NextButton to="/compute/kubernetes/how-to/create-kosmos-cluster/">How to create a Kosmos cluster</NextButton>
```

File: compute/kubernetes/api-cli/managing-load-balancer-ips.mdx [hunk 2]
```diff
@@ -13,7 +13,7 @@ dates:
   
 </Message>
 
-## Reserving a load b balancer IP using the API 
+## Reserving a load balancer IP using the API
```
Comment (ldecarvalho-doc) [inline compute/kubernetes/api-cli/managing-load-balancer-ips.mdx:16]:
```suggestion
## Reserving a Load Balancer IP using the API
```

File: compute/functions/api-cli/fun-uploading-with-serverless-framework.mdx [hunk 3]
```diff
@@ -225,3 +225,7 @@ This removes all your functions, as well as your Scaleway Functions' namespace.
   Your Scaleway Functions Registry namespace is not automatically removed. You must delete it manually either via the Scaleway console, or Scaleway's Functions Registry API.
  
 </Message>
```
Comment (ldecarvalho-doc) [inline compute/functions/api-cli/fun-uploading-with-serverless-framework.mdx:227]: there seems to be an issue with the code here

Comment (RoRoJ) [reply compute/functions/api-cli/fun-uploading-with-serverless-framework.mdx:227]: Yes I thought that, but in VSCode it looks fine, and the page looks/works fine with a yarn develop...

**Review by ofranc — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**